---
sensor_msgs/PointCloud2 mask_cloud